sympy/logic/boolalg.py

```diff
@@ -855,7 +847,7 @@ class Not(BooleanFunction):
       ``true`` and ``false``.
 
     >>> from sympy import true
-    >>> ~True
+    >>> ~True  # doctest: +SKIP
```

Comment:
What's wrong with this doctest?

Reply:
It gives a warning on newer Python versions:
```
$ python -Werror
Python 3.12.0 (main, Oct 13 2023, 13:56:30) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> ~True
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
DeprecationWarning: Bitwise inversion '~' on bool is deprecated. This returns the bitwise inversion of the underlying int object and is usually not what you expect from negating a bool. Use the 'not' operator for boolean negation or ~int(x) if you really want the bitwise inversion of the underlying int.
```

Reply:
Oh that's nice. This is definitely a good thing for them to remove. I suppose we should add a small parenthetical about this to the docstring.